eth1_refactor/taskexecutor/task_executor.go

```diff
@@ -0,0 +1,27 @@
+package taskexecutor
```

Comment:
thought: still not sure, but it might be good to move this package to be next to validatorcontroller, or just import it from there.